Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl

```diff
@@ -0,0 +1,1264 @@
+
+<#LuirccLóegaire>
+    a foaf:Person;
+    irishRel:genName "Luircc Lóegaire";
+    irishRel:nomName "Lorcc Lóegaire";
+    owl:sameAs <#LóegaireLorcc>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#Lugaid>
+    a foaf:Person;
+    irishRel:nomName "Lugaid";
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#Sétna>
+    a foaf:Person;
+    irishRel:nomName "Sétna";
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#CoelCobthach>
+    a foaf:Person;
+    irishRel:nomName "Coel Cobthach";
+    owl:sameAs <#CobthachCóel>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#Augaine>
+    a foaf:Person;
+    irishRel:nomName "Augaine";
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#AugaineMár>;
+    owl:sameAs <http://example.com/LL/comuammand_na_n-genelach.ttl#UgaineMár>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<http://www.vanhamel.nl/codecs/Ní_celt_ceis>
+    prov:wasAttributedTo <#Find>;
+    rdfs:comment "amal ad-fét Find m. Rossa isin fursunnud do-rigni".
+
+<#Find>
+    a foaf:Person;
+    irishRel:nomName "Find";
+    rel:childOf <#Rossa>;
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#Find>.
+
+<#Rossa>
+    a foaf:Person;
+    irishRel:nomName "Ros";
+    irishRel:genName "Rossa";
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#RossaRuaid>.
+
+<#Labraid-b9133180>
+    a irishTitle:Rí ;
+    irishRel:nomName "Labraid";
+    owl:sameAs <#Labraid-4d52ed70>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Ní_celt_ceis>;
+    rdfs:comment "Is iarum ro gab rígi n-Érenn amal ad-fét Find m. Rossa isin fursunnud do-rigni. Gabais dano rígi fer n-Alban & do-breth Muiriath ingen Scoriath ríg Fer Morcc dó ut poeta dixit".
+
+<#Muiriath>
+    a foaf:Person;
+    irishRel:accName "Muriath";
+    irishRel:nomName "Muriath";
+    foaf:gender "female";
+    owl:sameAs <http://example.com/LL/lagin.ttl#Moriath>;
+    rel:childOf <#Scoriath>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Ní_celt_ceis>;
+    rel:spouseOf <#Labraid-b9133180>.
+
+<#Scoriath>
+    a irishTitle:Rí ;
+    irishRel:genName "Scoriath";
+    irishRel:nomName "Scoriath";
+    rdfs:comment "ríg Fer Morcc".
+
+<http://www.vanhamel.nl/codecs/Nuadu_Necht_n%C3%AD_d%C3%A1mair_anflaith>
+    prov:wasAttributedTo <#FinnFili>.
+
+<#FinnFili>
+    a foaf:Person;
+    irishRel:nomName "Finn Fili";
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#Find>.
+
+<http://www.vanhamel.nl/codecs/Lug_sc%C3%A9ith>
+    prov:wasAttributedTo [
+      a foaf:Person;
+      foaf:title "poeta"
+    ].
+
+<#Áine>
+    a foaf:Person;
+    irishRel:genName "Áine";
+    rel:parentOf [
+      a foaf:Person;
+      owl:sameAs <#LabraidLoingsechMóen>
```

Comment:
The poem simply refers to "mac Áine" (this is Gaelic panegyric: people get referred to via allusions to their places in various networks). In context, this is clearly Labraid Loingsech, son of Ailill Áine (see para. 23).